eventmesh-connectors/eventmesh-connector-jdbc/src/main/java/org/apache/eventmesh/connector/jdbc/connection/JdbcConnection.java

```diff
@@ -46,13 +47,9 @@ public class JdbcConnection implements AutoCloseable {
     private static final String STATEMENT_DELIMITER = ";";
 
     private final JdbcConfig jdbcConfig;
-
-    private volatile Connection connection;
-
     private final InitialOperation initialOperation;
-
     private final ConnectionFactory connectionFactory;
-
+    private volatile Connection connection;
     private JdbcDriverMetaData jdbcDriverMetaData;
```

Comment:
There is no need to change field positions and blank lines are missing.

Also, the `@Slf4j` annotation on L37 of this class should be located below the javadoc.